include/grpc/event_engine/event_engine.h

```diff
@@ -501,7 +501,8 @@ std::shared_ptr<EventEngine> CreateEventEngine();
 /// Set the default EventEngine instance, which will be used throughout gRPC
 ///
 /// gRPC will hold a ref to this engine. For your engine to be shut down, you
-/// must call \a ShutdownDefaultEventEngine at the end of your program.
+/// must call \a ShutdownDefaultEventEngine at the end of your program, or
+/// \a SetDefaultEventEngine(nullptr).
```

Comment:
The wording here seems misleading.  Calling `SetDefaultEventEngine(nullptr)` doesn't shut anything down; it just means that gRPC drops its ref.

Reply:
The engine will never shut down unless one or the other is called. In other words: For it to ever shut down, one or the other must be called.

Reply:
I agree that this comment is strictly correct, but my concern is that users won't read it that way.  I think that they may read "if you need it to shut down, you must do X or Y" as "doing X or Y will shut it down".  The first is correct, but the second is not -- I think the reality here is that Y is necessary but not sufficient to shut down.

I suggest the following alternative wording:

"""
gRPC will hold a ref to this engine until either \a ShutdownDefaultEventEngine() is called or \a SetDefaultEventEngine() is called again with a different value.  Passing a value of nullptr will cause gRPC to drop the ref it was holding without setting it to a new one.
"""

Reply:
Documentation updated.